### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by RMaschkiwitz**
I forked from the main project into my own space before getting access to the gantner-nv space. I updated the dependencies to be compatible with our feature flag development.

### CODE REVIEW

**Comment by dietermijle** [comment]
Please resolve the conflicts @RMaschkiwitz

File: src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs [hunk 1]
```diff
@@ -459,5 +459,7 @@ public Task<UntagQueueResponse> UntagQueueAsync(UntagQueueRequest request, Cance
         }
 
         public IClientConfig Config => amazonSqsToBeExtended.Config;
+
+        public ISQSPaginatorFactory Paginators => throw new NotImplementedException();
```
Comment (dietermijle) [inline src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs]: Is this being used anywhere? Don't we need to implement this?

**Review by dietermijle — COMMENTED** [review]
Why didn't we create the PR towards the official lib?

File: src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs (hunk 1 above)
Comment (RMaschkiwitz) [reply src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs]: I don't see it being used anywhere and after searching for quite some time, I can't find good documentation on it or implementation that uses it.
The only thing I can find is that some of the methods like ListDeadLetterSourceQueues support pagination by setting the MaxResults property in the request and if it isn't set, up to 1000 results will be retrieved.

Comment (RMaschkiwitz) [reply src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs]: I have now tried just returning the same property from the injected IAmazonSQS object.

**Review by dietermijle — APPROVED** [review]

**Comment by dietermijle** [comment]
@robvanpamel please review this as well!

**Review by almsyx — APPROVED** [review]

**Pull request merged**